Source/Android/app/src/main/java/org/dolphinemu/dolphinemu/features/settings/model/FloatSetting.kt

```diff
@@ -10,7 +10,9 @@ enum class FloatSetting(
 ) : AbstractFloatSetting {
     // These entries have the same names and order as in C++, just for consistency.
     MAIN_EMULATION_SPEED(Settings.FILE_DOLPHIN, Settings.SECTION_INI_CORE, "EmulationSpeed", 1.0f),
-    MAIN_OVERCLOCK(Settings.FILE_DOLPHIN, Settings.SECTION_INI_CORE, "Overclock", 1.0f);
+    MAIN_OVERCLOCK(Settings.FILE_DOLPHIN, Settings.SECTION_INI_CORE, "Overclock", 1.0f),
+    GFX_CC_GAME_GAMMA(Settings.FILE_GFX, Settings.SECTION_GFX_COLOR_CORRECTION, "GameGamma", 2.35f),
+    GFX_CC_HDR_PAPER_WHITE_NITS(Settings.FILE_GFX, Settings.SECTION_GFX_COLOR_CORRECTION, "HDRPaperWhiteNits", 200.0f);
```

Comment:
I just noticed, to allow HDR to engage itself, `GFX_ENHANCE_HDR_OUTPUT` also needs to be exposed and set to on.
Then on Vulkan it will automatically engage if scRGB (R16G16B16A16F linear Rec.709/sRGB) is found to be between the allowed output formats. To tell whether the game is running in HDR or not, you can change the HDR paper white brightness and see if it's changing the output brightness.

Reply:
Ah ok, that's why I was so confused yesterday. I'll have that ready soon.

Reply:
Just for the record, the current HDR implementation does not work with android and settings related to it were removed from this PR.